src/app/models/folder-vo.ts

```diff
@@ -66,12 +67,16 @@ export class FolderVO
 
   // Thumbnails
   public thumbStatus;
-  public thumbURL200;
-  public thumbURL500;
-  public thumbURL1000;
-  public thumbURL2000;
+  public thumbURL200: string;
+  public thumbURL500: string;
+  public thumbURL1000: string;
+  public thumbURL2000: string;
   public thumbDT;
 
+  // New thumbnails
+  public thumbnail256: string;
+  public thumbnail256CloudPath: string;
```

Comment:
I'm actually not sure what the `CloudPath` property should be used for, if it should be used at all.